moped-editor/src/views/projects/projectView/FundingAmountIntegerField.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { TextField } from "@material-ui/core";
+
+/**
+ * MUI TextField wrapper that limits input to 0-9 for project funding amount
+ * and handles editors pasting a number with a decimal Ex. $86,753.09 -> 86753
```

Comment:
🎶 🎶

Reply:
This makes me so happy! I'm glad you brought up the 🥚 in retro because this flew right over my head. 😄